app/controllers/merchant/items_controller.rb

```diff
@@ -39,9 +39,9 @@ def activate
     item = Item.find(params[:id])
     item.toggle(:active?).save
     if item.active?
-      flash[:notice] = "Yer item is active! Now go get the booty.."
+      flash[:success] = "Yer item is active! Now go get the booty.."
     else
-      flash[:notice] = "Yer item has scurvy. Inactive!"
+      flash[:error] = "Yer item has scurvy. Inactive!"
```

Comment:
I think this one can be a success, because the user has clicked "Deactivate" and the change was successfully made.